src/main/java/api/authorization/AzureManagementAuthorizationManager.java

```diff
@@ -0,0 +1,29 @@
+package api.authorization;
```

Comment:
Why we need 2 authorization manager and cant use a generic one with relevant config?
That way we can easily add more api without the need to create authorization manager for each new api

Reply:
For each authentication method we need to save additional information like token expire time and we want to have only one authoriser initialised  per authentication address.
I can create a Upper level manager class that will contain the token and the expire time in a map and access them by the authentication address (will require adding to each api a method that will return its authentication address so we can get the correct token).